File: Raspberry Pi/src/main/angular/app/JoystickControl.ts
// Класс, который позволяет хранить и передавать значения от джойстика на сервер.

export class JoystickControl {
    public distance: number = 0;
    public angle: number = 0;

    reset() :void {
        this.distance = 0;
        this.angle = 0;
    };
}

File: Raspberry Pi/src/main/java/ru/tsirkunov/afelbot/JoystickControl.java
package ru.tsirkunov.afelbot;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

/**
 * Класс для передачи данных, связанных с джойстиком.
 *
 * @author Циркунов Виталий Андреевич
 */
@ToString
public class JoystickControl {

    /**
     * Угол отклонения джойстика от крайнего правого положения в радианах.
     */
    @Getter
    @Setter
    private double angle;

    /**
     * Расстояние от центра джойстика до ручки (0 до 100)
     */
    @Getter
    @Setter
    private double distance;

}
